### DISCUSSION TITLE
How can I made a notebook readonly?

### DISCUSSION THREAD

**Discussion by epugh** [jupyterlite/jupyterlite#1320] [Q&A]
I am shipping a set of notebooks in a `/examples` directory, and I'd like them to be readonly, so the user needs to make a copy in order to play with them.  Is that possible?

**Comment by jtpio** [comment]
Thanks @epugh.

This is likely not possible at the moment with the default file browser. However as a workaround you may want to distribute the notebooks via a custom drive. For example via `jupyterlab-github` which is enabled on the demo site: https://jupyterlite.readthedocs.io/en/latest/try/lab.

Although currently the UX for copying files between drives is not great, but may eventually be improved: https://github.com/jupyterlab/jupyterlab/issues/7755

Hopefully this will be improved by https://github.com/jupyterlite/jupyterlite/issues/1041 (and related issues) at some point.
